docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
+and maintain CPython's documentation through curation,
+internationalization, and special projects. Its focus will also include
+tutorials, how-to documents, the devguide and other documentation in support
+of the PSF's mission to expand the global Python userbase.
+Quality and effectiveness will be key goals as we build a
+diverse team of community members, documentarians, and core developers.
+
+Things the workgroup could do:
+
+- Develop the governance model for docs
+- Be the steering council for docs
+- Be the authors of doc guidelines
+- Be the authors/editors of docs
+- Request grant funding from PSF, if needed
+- Form an editorial board for docs including core devs, educators, and
+  documentarians in the Python community
+
+Active Time
+-----------
+
+This workgroup will be active indefinitely while supported by the Python
+Steering Council.
+
+Core Values & Internal Governance
```

Comment:
This section is just on values and removing members for behavioural violations, not internal governance. Perhaps:

```suggestion
Values & Behaviour
```

Reply:
This is part of the official [PSF Charter Template](https://wiki.python.org/psf/Example%20PSF%20Workgroup%20Charter) that per that page, it "must" include, so I don't think its wise to change it here.

Reply:
We already break those rules by having the charter listed here and not on the wiki ;-)

A

Reply:
AFAIK, that's just because the charter is a draft and the group isn't officially fully spun up yet, since it isn't listed on any official PSF sources that I can find. It is stated as a hard limitation ("must") in terms of including sectoins, so I don't see why we should break with that due to one community member's personal verbiage preference, given the description is still quite apt for the contents.